### DISCUSSION TITLE
Step2 Test with Actions is not updating for me!

### DISCUSSION THREAD

**Discussion by meeowngit** [skills/.github#279] [Q&A] [answered]
I messed with **Step 2: Fix the test**  the test-with-actions skills continuously, cause I earlier also made these changes and delete the repo again and start it again and remove this `.`   (as you can see in the given below snippets)

**before**

```
- name: Run markdown lint
  run: |
    npm install remark-cli remark-preset-lint-consistent
    npx remark . --use remark-preset-lint-consistent --frail
```

**after**

```
- name: Run markdown lint
  run: |
    npm install remark-cli remark-preset-lint-consistent
    npx remark --use remark-preset-lint-consistent --frail
```

And also I couldn't get this part of this [activity](https://github.com/meeowngit/skills-test-with-actions/tree/ci#keyboard-activity-fix-the-test)

Here is my [pull request](https://github.com/meeowngit/skills-test-with-actions/pull/1)

It doesn't allowing me to move to next **Step 3**  :disappointed: 

Don't know what to do?
Please help!

**Comment by meeowngit** [comment, marked as answer]
Hi all! I finally ended with my new **Step: 3** 
I have to modify `_` to `*` for italic fonts in `resume.md`

All that error would go away NOW! :smile:

**Comment by Ninad-Golatkar** [comment]
@meeowngit, thanks for the question, I was stuck there
First, we have to correct the indentation, as shown by @meeowngit above (the editor also helps with it through color change)
It took a lot of time for the next error (the one in 'resume.md' file) to show up (or me to notice it)
Then, we have to modify `*` on line 9 of resume.md before and after the word Git to `_ ` for italic fonts
Or the other way round as @meeowngit mentioned, change all `_` to `*`
I think the error is a consistency error, so we either use `_` or `*`, but consistently and everywhere in the file.
